sample/authui-react/package.json

```diff
@@ -13,9 +13,9 @@
     "@types/react-router": "^5.1.8",
     "@types/react-router-dom": "^5.1.5",
     "bootstrap": "^4.5.0",
-    "firebase": "^9.1.3",
+    "firebase": "^9.8.3",
     "firebaseui": "^6.0.0",
-    "gcip-iap": "^1.0.1",
+    "gcip-iap": "file:../..",
```

Comment:
Change to the latest version 2.0.0

Reply:
Made it ^1.0.1 here and in angular folder